app/services/laa_reference_creator.rb

```diff
@@ -10,15 +10,22 @@ def initialize(defendant_id:, user_name: nil, maat_reference: nil)
   end
 
   def call
-    create_laa_reference!
-    MaatLinkCreatorWorker.perform_async(Current.request_id, laa_reference.id)
+    persist_laa_reference!
```

Comment:
I'm interested in the argument for changing the name of this method.

Reply:
We're inside a class called `LaaReferenceCreator`, which does a bunch of things, so I wanted to be a bit more specific here.